modules/3d-tiles/src/tz3/tz3-parser.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  FileProvider,
+  HashElement,
+  cdSignature as cdHeaderSignature,
+  generateHashInfo,
+  parseHashFile,
+  parseZipCDFileHeader,
+  parseZipLocalFileHeader,
+  searchFromTheEnd
+} from '@loaders.gl/zip';
+import {TZ3Archive} from './tz3-archive';
+
+/**
+ * Creates 3tz file handler from raw file
+ * @param fileProvider raw file data
+ * @param cb is called with information message during parsing
+ * @returns 3tz file handler
+ */
+export const parse3tz = async (
+  fileProvider: FileProvider,
+  cb?: (msg: string) => void
```

Comment:
Callback parameters is not something we have been using in loaders.gl. 
Perhaps passing in a `log` instance could solve the problem here?